packages/muelu/src/Misc/MueLu_CoordinatesTransferFactory_decl.hpp

```diff
@@ -95,11 +96,21 @@ namespace MueLu {
   ----------|--------------|------------
   | Coordinates | CoordinatesTransferFactory   | coarse level coordinates
 */
-  template<class Scalar = DefaultScalar,
-           class LocalOrdinal = DefaultLocalOrdinal,
-           class GlobalOrdinal = DefaultGlobalOrdinal,
-           class Node = DefaultNode>
-  class CoordinatesTransferFactory : public TwoLevelFactoryBase {
+  template <class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
```

Comment:
Why is this needed?

Reply:
It's how our *_kokkos factories are implemented. If we prefer that, I can try to make that the usual SC,LO,GO,NO template args.

Reply:
Is it that `Kokkos::Compat::KokkosDeviceWrapperNode<DeviceType>` is a specialization of `Node`?

Reply:
Maybe that pattern is from before the days of KokkosCompat? Lemme just remove that and see if I see any impact.

Reply:
Could it be that the `decl` only is getting included elsewhere, and that provides the forward declaration?

Reply:
Seems to work just fine locally. Let's see if the AT finds anything.